services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/_scheduler_base.py

```diff
@@ -688,41 +694,50 @@ async def apply(
                         f"{pipeline_result=}",
                     )
             except PipelineNotFoundError:
-                _logger.warning(
-                    "pipeline %s does not exist in comp_pipeline table, it will be removed from scheduler",
+                _logger.exception(
```

Comment:
TIP: use `create_troubleshootting_log_kwargs` helper by default to log exceptions
```python

except PipelineNotFoundError as exc:
    _logger.exception(
        **create_troubleshootting_log_kwargs(
           "pipeline {project_id} is missing from `comp_pipelines` DB table, something is corrupted. Aborting scheduling" ,
            error=exc,
            error_context={"project_id": project_id, ...},
        )
    )
```

Reply:
why I need to get the exc with this guy? logger.exception gets it automatically? just a question.